src/components/Header/Header.js

```diff
@@ -2,8 +2,10 @@ import "./Header.css";
 
 function Header() {
   return (
-    <div className="Header__styling">
-      <h1>Harry Potter</h1>
+    <div>
+      <div class="Header__styling">
+        <h1>Harry Potter</h1>
+      </div>
     </div>
```

Comment:
Why do you have so many `div` elements here? And why not a semantic `header` element? Also `Header__styling` is not a good class name. Better would be just `Header`.